client/app/components/partials/SampleData.vue

```diff
@@ -72,15 +82,20 @@
       </sample-data-file>
     </v-flex>
 
-    <v-flex xs4   id="sample-selection">
-      <v-autocomplete
-        v-bind:class="samples == null || samples.length == 0 ? 'hide' : ''"
-        label="Sample"
-        v-model="sample"
-        :items="samples"
-        @input="onSampleSelected"
-        hide-details
-      ></v-autocomplete>
+    <v-flex xs8   id="sample-selection">
```

Comment:
It would be better to put the user message in its own <v-flex xs4> so that the width of the sample dropdown remains constant whether the user message shows or not.  Right now, since the user message shares the same div as the autocomplete, the width of the autocomplete shrinks when the user text shows.

Notice it is wider when the message doesn't show:
<img width="666" alt="Screenshot 2023-08-12 at 12 56 33 PM" src="https://github.com/iobio/gene.iobio/assets/8420103/7b58fa93-f595-448d-b4c1-d1e3c3718c56">

And narrower when the message does show:
<img width="654" alt="Screenshot 2023-08-12 at 12 56 43 PM" src="https://github.com/iobio/gene.iobio/assets/8420103/ce2658f4-15ff-495e-b718-daf6c2e6e902">

Reply:
@tonydisera you have good attention to detail! I changed the display.